api/src/main/resources/liquibase.xml

```diff
@@ -27,7 +27,7 @@
             <column name="uri" type="varchar(255)" />
             <column name="object" type="varchar(1000)" />
             <column name="category" type="varchar(255)" />
-            <column name="date_created" type="timestamp" />
+            <column name="date_created" type="timestamp" defaultValueDate="CURRENT_TIMESTAMP"/>
```

Comment:
We shouldn't change previous migrations, as the build might fail due to the wrong checksum. I guess you didn't want to put it into the commit anyway ;)

Reply:
You are right, thank you for noticing it!